app/addons/paynl_addon/paynl/classes/Pay/Api/Start.php

```diff
@@ -242,7 +247,7 @@ protected function _getPostData() {
         }
 
         
-        $data['ipAddress'] = $_SERVER['REMOTE_ADDR'];
+        $data['ipAddress'] = $this->_ipaddress;
```

Comment:
No check if the ipaddress is set?
Maybe add an if here, if its not set, retreive it from $_SERVER['REMOTE_ADDR'] like it was.

Reply:
done